www/src/templates/template-api-markdown.js

```diff
@@ -74,6 +75,12 @@ const mergeFunctions = (data, context) => {
   return mergedFuncs
 }
 
+const containerStyles = {
```

Comment:
Most of these styles are copied from `template-docs-markdown` right? It would be helpful to refactor things to that we don't repeat the same style definitions across two files.

We could make a component called something like `<DocsMarkdownPage>` and put styles common between the two there.

Reply:
Yep I thought the same! I'll do it 😄